### DISCUSSION TITLE
Cannot apply for academic reverification without school-issued email ID

### DISCUSSION THREAD

**Discussion by adityansx** [community/community#50341] [GitHub Education]
Hey all, I recently received a mail to reverify my academic affiliation. I tried to complete the process and couldn't even make past the page where you fill your university's name without a school-issued email address. It errors out with the message "We require applicants to use a school-issued email address to apply."

It's really common in India for most of the colleges and universities to not provide an email address to their students. If there's anything I can do to resolve this issue, I'd like to know.

**Comment by mdjvd** [comment]
I have the same problem in Iran.

**Reply by adityansx** [reply]
How long has this been happening to you ? Were you able to clear the verification the first time you applied ?

**Reply by mdjvd** [reply]
Yes, I was verified the first time with no problem, about 6 months ago. Yesterday my friend and I received an email telling us we must reverify, so I did so. But the problem is when I select my existing email and my university, it shows a message and says I must use a school-issued email but my university doesn't provide us with one. Now I am totally unable to reverify and I don't know what to do.
